scripts/dist-tag.ts

```diff
@@ -60,11 +66,12 @@ Usage:
   if (!tag) {
     throw new Error('Tag must be non-empty, for example: latest, next, v10-lts, etc');
   }
+  const registry = registryArg ?? wombat;
```

Comment:
The wombat registry will not work in this case, I think.
@mhevery tried it once and ran into authentication errors.
It has to be the default NPM registry.

Also, since registry is non-optional, we don't need nullish-coalescing.

Reply:
Tested this and Wombat does work with the `npm dist-tag` command.

`--registry` is optional, defaulting to Wombat, so I believe this is necessary.